src/substrate/types.ts

```diff
@@ -138,8 +137,9 @@ interface IEvent {
  */
 export interface INewSession extends IEvent {
   kind: EventKind.NewSession
-  validators: Array<ValidatorId>;
-  exposure: Array<Exposure>;
+  active: Array<_AccountId>
```

Comment:
We should be stringifying these AccountIds as we do in all the other event interfaces. We also might want to marshal the Exposure struct for serialization purposes -- see my comment on the other PR.

Reply:
I have used a dictionary approach with active validators and their respective exposures in the new commit, lmk, if there's something I should change.